### PR TITLE
Feature#32 player functionality

### LINKED ISSUE #32: Player functionality
# REQUIREMENTS
Add player functionality

### PR DESCRIPTION
**Pull request by MauricioNC**
# Description
To show the video player, we send the video title to the controller and
redirect to player view.

- Player controller:
 - The controller catch the param from home view and make a request to
   get the respective video.
 - Once obtained the video, the controller iterate response and get the
   author (the user that uploaded the video) and get all the other videos
   to show them as a videos related.
 - Also, we get all the events and get the author of that event to show
   them.
  
  ## Screenshots 
image
image

### CODE REVIEW

**Review by xFerchoVJ — APPROVED** [review]
LGFM

**Review by LuisGomez89 — APPROVED** [review]

File: EasyLearn/app/assets/stylesheets/player.css [hunk 1]
```diff
@@ -1,87 +1,94 @@
...
+  border-bottom: 1px solid #F78528;
+  color: rgb(198, 198, 201);
+  margin-bottom: 5rem;
+  padding-bottom: 15px;
+  width: 50%;
+}
+
+.card-deck .card {
+  width: 22.8%;
+}
```
Comment (JuanequeX) [inline EasyLearn/app/assets/stylesheets/player.css]: Add a blank line here

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: EasyLearn/app/assets/stylesheets/player.css (hunk 1 above)
Comment (MauricioNC) [reply EasyLearn/app/assets/stylesheets/player.css]: Done.

**Review by javs1 — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**